terraform/sentinel/README.md

```diff
@@ -0,0 +1,317 @@
+- GitHub account as this demo uses GitHub Actions to automate the DNS change management process. If you would like to set up a GitOps workflow with GitHub Actions. Alternatively, you can only do the demo locally or directly through Terraform Cloud.
+- [Terraform Cloud](https://app.terraform.io/signup/account) account and an API token. You can create a new token in the [Terraform Cloud user settings](https://app.terraform.io/app/settings/tokens). Please take note of the token as you will need it later. This is not optional as you will need to use Terraform Cloud to run the Sentinel policy.
+
+## Getting Started
+
+There are two parts to this demo. The first part is to set up and run the demo locally with Terraform Cloud. The second part is to set up GitHub Actions to automate the DNS change management process.
+
+### Part 1: Run the demo locally with Terraform Cloud
+
+#### Step 1: Clone the repository
+
+Clone this repository to your local machine:
+
+```bash
+git clone git@github.com:dnsimple/dnsimple-api-examples.git
+cd dnsimple-api-examples/terraform/sentinel
+```
+
+#### Step 2: Setup Terraform Cloud
+
+1. Create a new workspace in Terraform Cloud. Select the "API-driven workflow" option.
+2. Next, we want to authenticate your local environment with Terraform Cloud. We will do this by setting environment variables as follows:
+
+```bash
+export TF_CLOUD_ORGANIZATION="<your organization name>"
+export TF_WORKSPACE="<your Terraform Cloud workspace name>"
+export TF_TOKEN_app_terraform_io="<your Terraform Cloud API token>"
+```
+
+Alternatively, you can authenticate with Terraform Cloud by running `terraform login` and following the prompts. If you take this route you will also need to add the following to your `main.tf` file:
+
+```hcl
+terraform {
+  cloud {
+    organization = "<your organization name>"
+
+    workspaces {
+      name = "<your Terraform Cloud workspace name>"
+    }
+  }
+}
+```
+
+#### Step 3: Setup the domains we will be using
+
+1. Begin by copying the `terraform.tfvars.example` file to `terraform.tfvars`:
+
+```bash
+cp terraform.tfvars.example terraform.tfvars
+```
+
+Fill out the `terraform.tfvars` file with the information for the domains you will be using. You will also create these variables at the workspace level in Terraform Cloud.
+
+2. Next, we will import the domains into our Terraform state. Run the following commands:
+
+```bash
+terraform init
+```
+
+```bash
+terraform import dnsimple_registered_domain.domain_a <domain_a>
+terraform import dnsimple_registered_domain.domain_b <domain_b>
+```
+
+#### Step 4: Setup the Sentinel policy
+
+Create a new policy in Terraform Cloud. Copy the contents of `policies.sentinel` into the policy. You will also need to create a policy set and attach the policy to the policy set. When creating the policy set, you will should select the workspace you created in Step 2, otherwise, the policies will be applied to all workspaces in your organization.
```

Comment:
```suggestion
Create a new policy in Terraform Cloud. Copy the contents of `policies.sentinel` into the policy. You will also need to create a policy set and attach the policy to the policy set. When creating the policy set, select the workspace you created in Step 2, otherwise the policies will be applied to all workspaces in your organization.
```